product-service/src/services/index.ts

```diff
@@ -0,0 +1,13 @@
+import '../config/config';
+
+export const findProductByTitle = (process.env.NODE_ENV === 'test') ?
```

Comment:
If you are wondering how this could have been avoided, I suggest reading about the principle of **Dependency inversion**. By following this principle, you can easily mock some functional blocks that are not checked in the test examples.
You can read about it here, for example - https://medium.com/webbdev/solid-4ffc018077da.